tests/e2e/test_main.py

```diff
@@ -762,6 +764,34 @@ def test_data_cp_from_cloud_to_local(
             assert expected_archive.is_file()
 
 
+@pytest.mark.parametrize("bucket", [GCP_BUCKET, AWS_BUCKET])
+@pytest.mark.parametrize("archive_extension", ["tar.gz", "tgz", "zip", "tar"])
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="Windows path are not supported yet + no utilities on windows",
+)
+def test_data_cp_from_cloud_to_local_compress(
```

Comment:
Could we also cover the case when one wants to compress artifacts, which are located on storage?

E.g. we have a folder with files on storage, which we want to compress and keep the archive on storage. (in the future, I think, we will need to extend this case with pushing into the cloud).

Reply:
I don't think data ingestion/extraction is applicable to storage ⇔ storage copying. It's explicitly prohibited in the code now and there's no such functionality at the moment.

Reply:
True, my fault. Probably, in the future, we will have such a requirement, but not now. Thanks for the explanation!